CHANGELOG.md

```diff
@@ -21,8 +22,6 @@
 * Node: Added FUNCTION DELETE command ([#1990](https://github.com/valkey-io/valkey-glide/pull/1990))
 * Node: Added FUNCTION FLUSH command ([#1984](https://github.com/valkey-io/valkey-glide/pull/1984))
 * Node: Added ZMPOP command ([#1994](https://github.com/valkey-io/valkey-glide/pull/1994))
-
-#### Fixes
```

Comment:
Shouldn't we still have the "Fixes" heading here?

Reply:
I must've made a mistake while resolving merge conflicts if that's showing up somehow